src/EditorFeatures/Next/IntelliSense/Completion/Presentation/FilteredRoslynCompletionSet.cs

```diff
@@ -0,0 +1,239 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
This is identical to `RoslynCompletionSet.cs` in EditorFeatures.  Not sure the right way to factor it yet.  Kind of wish C# had mix-ins right now :)

Maybe @CyrusNajmabadi has ideas.